ev3dev2/stopwatch.py

```diff
@@ -5,7 +5,7 @@
 try:
     import datetime as dt
     micropython = False
-except ImportError:
+except Exception:
     import utime
     micropython = True
```

Comment:
isn't there already a function we have to check this? And/or a uPy builtin?

Reply:
ah you are right, we have
```
def is_micropython():
    return sys.implementation.name == "micropython"
```

Will use that